### ISSUE TITLE
Fix Linkedin bug in Team Roster

### ISSUE THREAD

**Issue by itsevalieu** [hackforla/shared-housing#212]
Get the LinkedIn icons/links, include the material-ui icons. You'll need to install material-ui icon packages.

**Comment by wsbuck** [comment]
@kennethlee I assigned you to this. Let me know if you run into any trouble again. I know you mentioned an issue with the icons before. I thought it had to do with material-ui/icons being a seperate package from material-ui/core but it looks like we already have the icons package downloaded.

**Comment by kennethlee** [comment]
@wsbuck Yeah I tried using other icons from the m-ui/icons lib and they worked just fine, it was specifically only the linkedin icon that didn't work (strange).

Anyway, I've been troubleshooting some frustrating dev environment issues on my machine since yesterday but will get to this after that is ironed out. Feel free to assign me others in the meantime; otherwise, I'll take a look at the outstanding issues and pick a couple up later.

**Comment by karlmageddon** [comment]
It might have something to do with the version of m-ui/icons we currently have. Maybe updating it will yield the LI icon?

**Comment by kennethlee** [comment]
@wsbuck @karlmageddon reached out to me wondering if he could take a crack at making aesthetic improvements to the team roster; perhaps we should assign this issue to him instead?

**Comment by ExperimentsInHonesty** [comment]
@karlmageddon are you still working on this, and if so what has been done?  Sent slack message as well.
